partners_api/partners_api_tests/yandex_tests.cpp

```diff
@@ -0,0 +1,55 @@
+#include "testing/testing.hpp"
+
+#include "partners_api/yandex_api.hpp"
+
+#include "geometry/latlon.hpp"
+
+#include "base/scope_guard.hpp"
+
+namespace
+{
+UNIT_TEST(Yandex_GetTaxiInfo)
+{
+  ms::LatLon const from(55.796918, 37.537859);
+  ms::LatLon const to(55.758213, 37.616093);
+  string result;
+
+  taxi::yandex::RawApi::GetTaxiInfo(from, to, result);
+
+  TEST(!result.empty(), ());
+}
+
+UNIT_TEST(Yandex_GetAvailableProducts)
+{
+  taxi::yandex::Api api;
+  ms::LatLon const from(55.796918, 37.537859);
+  ms::LatLon const to(55.758213, 37.616093);
+
+  taxi::yandex::SetYandexUrlForTesting("http://localhost:34568/partners");
```

Comment:
Again, suggest you to use UNIT_CLASS_TESTS, it will help you to get rid of all these MY_SCOPE_GUARDs.